src/main/java/com/minecolonies/proxy/ClientProxy.java

```diff
@@ -6,11 +6,14 @@
 import com.minecolonies.client.render.EmptyTileEntitySpecialRenderer;
 import com.minecolonies.client.render.RenderBipedCitizen;
 import com.minecolonies.client.render.RenderFishHook;
+import com.minecolonies.client.render.TileEntityScarecrowRenderer;
+import com.minecolonies.colony.CitizenData;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.CitizenData'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)